dev-python/pycryptodome/pycryptodome-3.4.3.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+# needed to fix the: No such file or directory: 'build/test1.c'
+DISTUTILS_IN_SOURCE_BUILD="1"
+
+inherit distutils-r1
+
+DESCRIPTION="Self-contained Python package of low-level cryptographic primitives"
+HOMEPAGE="https://github.com/Legrandin/pycryptodome"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+RESTRICT="test"
+
+LICENSE="BSD-2 Unlicense"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+DEPEND="!dev-python/pycrypto
+	dev-libs/gmp:0
+	dev-python/setuptools"
+RDEPEND="${DEPEND}"
```

Comment:
can you fine tune these? i.e.
```
RDEPEND="!dev-python/pycrypto
	dev-libs/gmp:0="
DEPEND="${RDEPEND}
	dev-python/setuptools"
```
Notice the subslot operator on gmp. Unless you have very good reasons, setuptools should not be in `DEPEND`